.github/styles/vocab.txt

```diff
@@ -207,6 +207,7 @@ onboarding
 pagerduty
 pageview
 parallelization
+params
```

Comment:
Prolly wrap this in a code block to avoid having to add it here, otherwise spell out parameters imo

Reply:
@Rugvip sounds good, updated